### PR TITLE
PUD-238 Specify latest Node.js and NPM versions

### PR DESCRIPTION
**Pull request by jonnywyatt**
https://dsdmoj.atlassian.net/browse/PUD-238

### CODE REVIEW

**Review by mikedunfordEE — APPROVED** [review]

**Review by mr-ken — APPROVED** [review]
🌟

File: .nvmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+14.17.3
```
Comment (philsegal) [inline .nvmrc:1]: More of a question, we have `14.17.3` in 3 places. I assume there isn't a way to have it in 1?

File: README.md [hunk 2]
```diff
@@ -4,12 +4,21 @@
...
 * manage-recalls-api - api service for managing the recall process
 
+### Node and NPM
+The required versions of Node.js and NPM are in package.json 'engines' field.
+Recommended that you use NVM to manage your Node versions. You can then upgrade to match the required versions with:
+```
+nvm install <version>
+nvm install-latest-npm
+```
+The Node version is also in .nvmrc so your shell can automatically switch to it when you change folders.
```
Comment (philsegal) [inline README.md:20]: What does this mean. Do we need to do anything in order to get our shell to switch automatically? (can/will)?

File: .nvmrc (hunk 1 above)
Comment (jonnywyatt) [reply .nvmrc:1]: yeah it's a good point :)
The source of truth could be package.json engines, and we extract to an env var that we can then use in docker config and scripts.
Not sure about the .nvmrc though. Maybe I'll try removing it actually and see if engines warning is enough.

File: README.md (hunk 2 above)
Comment (jonnywyatt) [reply README.md:20]: Maybe I'll take this and the .nvmrc out. It's just a convenience, not required
Yes you'd need to add something to your shell config. I'll take this out until I can improve it.

File: .nvmrc (hunk 1 above)
Comment (jonnywyatt) [reply .nvmrc]: I removed the .nvmrc

File: README.md (hunk 2 above)
Comment (jonnywyatt) [reply README.md]: Removed it

**Review by philsegal — APPROVED** [review]

**Pull request merged**